### ISSUE TITLE
Idea: configurable cssremedy builds

### ISSUE THREAD

**Issue by himedlooff** [mirisuzanne/remedocs#3]
Since Doxray breaks each rule up and since there is metadata attached to each one (category for example), you could hypothetically offer an interface that allows a user to include/exclude certain rules/categories. An example that comes to mind is maybe someone doesn't support legacy browsers, so they might want to use cssremedy but exclude all of the rules under the "legacy browsers" category.

**Comment by mirisuzanne** [comment]
Yeah, that's interesting - on the simpler end, we could just make that Json data file available, and let people do what they want with it.
